apps/finance/app/src/components/Transfers.js

```diff
@@ -58,6 +89,47 @@ class Transfers extends React.PureComponent {
       ...initialState,
     })
   }
+  handleDateRangeChange = selectedDateRange => {
+    this.setState({ selectedDateRange })
+  }
+  encodeDataToCsv = (data, tokenDetails) => {
+    const csvContent = [
+      'data:text/csv;charset=utf-8,Date,Source/Recipient,Reference,Amount',
+    ]
+      .concat(
+        data.map(
+          ({
+            date,
+            numData: { amount },
+            reference,
+            isIncoming,
+            entity,
+            token,
+          }) => {
+            const { symbol, decimals } = tokenDetails[toChecksumAddress(token)]
+            const formattedAmount = formatTokenAmount(
+              amount,
+              isIncoming,
+              decimals,
+              true,
+              { rounding: 5 }
+            )
+            return `${format(
+              date,
+              'dd/MM/yy'
+            )},${entity},${reference},${`${formattedAmount} ${symbol}`}`
```

Comment:
I think it might make sense to put the symbol in a separate column. This will allow for easy aggregation and filtering in most spreadsheet software. 

This is with [tad](https://www.tadviewer.com)
<img width="1276" alt="Screen Shot 2019-03-28 at 1 03 17 pm" src="https://user-images.githubusercontent.com/1992255/55156328-e48e9700-5159-11e9-9a68-2f0023f923b2.png">

CC: @dizzypaty @luisivan